src/main.cc

```diff
@@ -19,13 +19,19 @@ int main(int argc, char** argv) {
   spdlog::info(" - Host: {}", config.host);
   spdlog::info(" - Port: {}", config.port);
   spdlog::info(" - Num of Http Listener Threads: {}", config.listenerThreads);
+  spdlog::info(" - Keep-Alive Max Count: {}", config.keepAlive);
```

Comment:
```suggestion
  spdlog::info(" - Keep Alive Max Count: {}", config.keepAlive);
```
큰 상관은 없지만 얘만 - 인게 눈에 띈달까..?